src/tools/common/sss_tools.c

```diff
@@ -300,6 +263,49 @@ void sss_tool_usage(const char *tool_name, struct sss_route_cmd *commands)
     sss_tool_print_common_opts(min_len);
 }
 
+static int tool_cmd_init(struct sss_tool_ctx *tool_ctx,
+                         struct sss_route_cmd *command)
+{
+    int ret;
+
+    if (command->flags & SSS_TOOL_CMD_FLAG_NO_CONFDB) {
+        /* This tool does not need to connect to confdb.
+         * Nothing to do. */
+        return EOK;
+    }
+
+    /* Connect to confdb. */
+    ret = sss_tool_confdb_init(tool_ctx, &tool_ctx->confdb);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_CRIT_FAILURE, "Unable to open confdb [%d]: %s\n",
+                                   ret, sss_strerror(ret));
```

Comment:
Just a minor comment (and I know it was like this before ...), but could you align all parameters to the first one in case of new line?
I'd just go for:
```
    DEBUG(SSSDBG_CRIT_FAILURE, "Unable to open confdb [%d]: %s\n",
          ret, sss_strerror(ret));
```

(OTOH, as it was there before, feel free to ignore my comment ;-))